RP Cloud Overview and Security doc (#1504)
redpanda-data/documentation: docs/deploy/deployment-option/cloud/security/cloud-security-architecture/secrets.mdx

---
title: Secrets
---

<head>
    <meta name="title" content="Redpanda Cloud Secrets  | Redpanda Docs"/>
    <meta name="description" content="Learn how Redpanda Cloud manages secrets."/>
    <link rel="canonical" href="https://docs.redpanda.com/docs/deploy/deployment-option/cloud/security/cloud-security-architecture/secrets/" />
</head>

> Redpanda Cloud uses _dynamic secrets_ through IAM roles, which
> have policies defined by the actions and resources that a user (also

Reviewer: ```suggestion
Redpanda Cloud uses _dynamic secrets_ through IAM roles. These
have policies defined by the actions and resources that a user (also
```

known as a principal) strictly needs, following the principle of least privilege.

> Redpanda Cloud uses _static secrets_, stored in either the
> [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or
> [GCP Secret Manager](https://cloud.google.com/secret-manager) services.
> 
> Static secrets managed through Redpanda Console never leave their corresponding
> data plane account or network, and stay securely stored in AWS Secrets Manager or
> GCP Secret Manager, respectively.

Reviewer: ```suggestion
Redpanda Cloud also uses _static secrets_, stored in either the
[AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or
[GCP Secret Manager](https://cloud.google.com/secret-manager) services. Static secrets managed through Redpanda Console never leave their corresponding
data plane account or network. They stay securely stored in AWS Secrets Manager or
GCP Secret Manager.
```
